app/models/language.rb

```diff
@@ -58,55 +61,116 @@ def self.menu_options
   # Get a list of the top N contributors to a language's translations.
   # This is used by the app layout, so must cause mimimal database load.
   def top_contributors(num = 10)
-    user_ids = self.class.connection.select_rows(%(
-      SELECT user_id
-      FROM translation_strings
-      WHERE language_id = #{id} AND user_id != 0
-      GROUP BY user_id
-      ORDER BY COUNT(id) DESC
-      LIMIT #{num}
-    ))
+    id_select_manager = arel_select_top_contributors_ids(num)
+    # puts(id_select_manager.to_sql)
+    user_ids = self.class.connection.select_values(id_select_manager.to_sql)
+    # puts(user_ids.inspect)
     if user_ids.any?
-      user_ids = self.class.connection.select_rows(%(
-        SELECT id, login
-        FROM users
-        WHERE id IN (#{user_ids.join(",")})
-        ORDER BY FIND_IN_SET(id, '#{user_ids.join(",")}')
-      ))
+      user_select_manager = arel_select_top_contributors_in_order(user_ids)
+      user_ids = self.class.connection.select_rows(user_select_manager.to_sql)
+      # puts(user_ids.inspect)
     end
     user_ids
   end
```

Comment:
Consider rewriting this as AR, too:

  TranslationString.where(language: self).and.not(user_id: 0).group(:user_id).order("count(id)").limit(num).pluck(:user_id)

I would also consider refactoring the caller in order to avoid the FIND_IN_SET thing.  Instead, return just the user ids, let the caller grab the logins, either itself or via a private helper.